cppcache/src/ThinClientLocatorHelper.cpp

```diff
@@ -123,7 +124,8 @@ std::shared_ptr<Serializable> ThinClientLocatorHelper::sendRequest(
     auto conn = createConnection(location);
     auto data =
         m_poolDM->getConnectionManager().getCacheImpl()->createDataOutput();
-    data.writeInt(static_cast<int32_t>(1001));  // GOSSIPVERSION
+    data.writeInt(static_cast<int32_t>(1002));  // GOSSIPVERSION
+    data.writeInt(static_cast<int16_t>(125));   // GEODE 1.14.0
```

Comment:
Use `Version::getOrdinal()`. You will need to update the ordinal to 1.14.0. This also means any protocol changes from 1.0.0, the current value, to 1.14.0 need to be implemented.